### ISSUE TITLE
Icue becomes blocked by awakened Poe trade window

### ISSUE THREAD

**Issue by Tyneen** [SnosMe/awakened-poe-trade#497]
My last issue entry seemed to have gotten deleted, not sure why, but I’m sorry if I posted it incorrectly since I’m not someone who really knows how to use GitHub correctly. I saw some others having the same issue and tried to post in response.

Basically the window that pops up blocks all macro actions from being executed in Poe even when you refocus the game. It’s quite a disruptive problem as it keeps me from having this program open while playing, which was never a problem before the latest version.

I should mention this is also the stand alone Poe client, not steam.

This is not an Icue software issue as the last version 3.38.61 was put out on 03/15/2021 and this issue only started happening with awakened poe trade with its last update.

Just to be sure, I went back and downgraded to iCUE 3.33.246 and had the same issue, then updated to iCUE 4.16.194. Same issue. Reverting to the 2.13.1 version of awakened poe trade fixes this issue completely.

Also to be sure, I contacted Corsair support and they had me run a few tests by manually switching profiles, running macros from the mouse hardware and so on. Everything continued to fail with awakened Poe trade opened, but upon closing it all of their tests worked, so they assured me it is highly unlikely that it’s icue.

Is there a possible work around to this issue? A way to have an option to have the overlay work as it used to prior to the update? Or do you have something I can reference to Corsair that shows them there is a bug? Because I’m willing to raise the issue with them further if possible.

Thanks.

**Comment by SnosMe** [comment]
Duplicate of #477

As you can see in video https://github.com/SnosMe/awakened-poe-trade/issues/477#issuecomment-950763549 Windows generates events that icue can listen to.

> A way to have an option to have the overlay work as it used to prior to the update

No, the previous way was a hack, not officially documented by Windows and not working at all on Mac OS (for which I potentially want to add support also).

**Comment by Tyneen** [comment]
Installed the steam version of PoE and I do not have this issue at all, everything works completely fine with awakened poe trade open, mouse buttons pass through to the game after opening the overlay even multiple times.

So this seems 100% to be an issue on the stand alone client. I wouldn't know why though.

**Comment by SnosMe** [comment]
I don't have access to standalone, you can try to run this tool too https://docs.microsoft.com/en-us/windows/win32/winauto/accessible-event-watcher

with the settings

![image](https://user-images.githubusercontent.com/4292308/141021321-5b230e5d-1061-4a25-84e5-df2c94a05ca3.png)

![image](https://user-images.githubusercontent.com/4292308/141021378-d5f9298c-ce21-4fe9-9b95-278716da9f30.png)
